tests_config/run_spec.js

```diff
@@ -29,16 +39,26 @@ function run_spec(dirname, parsers, options) {
       filename[0] !== "." &&
       filename !== "jsfmt.spec.js"
     ) {
+      if (!semver.satisfies(PYTHON_VERSION, versionRange)) {
+        // Skip each test here with the snapshot name below to avoid obsolete
+        // snapshot failures.
```

Comment:
But they are obsolete, right?

Reply:
I tried to clarify the comment a bit.

The idea is that we run the _same_ tests for both Python 2 and Python 3. However, some of these tests can't be run for Python 3.

For those tests, we skip them when the system Python is Python 2 – but if we don't explicitly skip them like this, then they throw off "obsolete snapshot" failures.

Those failures would be correct if we were only dealing with Python 2, but it's more that they're not pertinent to the test suite as currently run.